src/AppM.purs

```diff
@@ -33,8 +36,14 @@ derive newtype instance monadEffectAppM :: MonadEffect AppM
 
 derive newtype instance monadAffAppM :: MonadAff AppM
 
+derive newtype instance monadLoggerAppM :: MonadLogger AppM
+
 instance monadAskAppM :: TypeEquals o Options => MonadAsk o AppM where
   ask = AppM $ asks from
 
 runAppM :: forall a. AppM a -> Options -> Aff a
-runAppM (AppM m) options = runReaderT m options
+runAppM (AppM m) options = runReaderT (runLoggerT m logMessage) options
+  where
+  logMessage { message }
+    | options.verbose = liftEffect $ Console.info message
```

Comment:
I wonder, should we allow for more levels at one point? Like only info, or info + error for instance. Not in this pr though, just an improvement for the future ^^

Reply:
I'm a bit confused here, it seems we don't really use the `level` system at all, and discard it entirely right? When, or if, we want to disambiguate info / warnings / errors, we might need to bring it back 😕 
Also, it means using the info/debug/etc... functions from https://pursuit.purescript.org/packages/purescript-monad-logger/1.3.1/docs/Control.Monad.Logger.Class has litterally no impact on the output?
Maybe we could also allow for a `DEBUG_LEVEL` env var instead? So that we don't a switch, and can use the full potential of the monad-logger lib?
(But to be honest I like the simplicity of your implementation, so it's just a question ^^)

Reply:
Yeah I think we should leverage the monad logger. I'm thinking of changing the flag to `verbosity` with default value `info` 🤔